Header/graph_color.h

```diff
@@ -0,0 +1,69 @@
+#ifndef _GRAPH_COLOR_H_
+#define _GRAPH_COLOR_H_
+
+#include <map>
+#include <stack>
+#include <locale>
+#include <vector>
+#include <string>
+#include <utility>
+#include <iostream>
+#include <queue>
+
+using std::queue;
+using std::map;
+using std::pair;
+using std::stack;
+using std::string;
+using std::vector;
+using std::cout;
+using std::cerr;
+using std::endl;
+
+namespace GraphColoring {
+    enum Algorithm {kNone,kDSATUR,kMCS,kTABUCOL,kHybrid,kLMXRLF,kHybridDSATUR};
```

Comment:
It's used to print the chromatic on the terminal. It tells you which algorithm ran. Did you still want me to delete it?